src/dune_patch/dune_patch.ml

```diff
@@ -15,18 +15,15 @@ include struct
 end
 
 let re =
-  let line xs = Re.seq ((Re.bol :: xs) @ [ Re.eol ]) in
-  let followed_by_line xs = Re.seq [ Re.str "\n"; line xs ] in
-  Re.compile
-  @@ Re.seq
-       [ Re.alt
-           [ line [ Re.str {|--- a/|}; Re.group (Re.rep1 Re.notnl) ]
-           ; line [ Re.str {|--- |}; Re.group (Re.rep1 Re.notnl) ]
-           ]
-       ; Re.alt
-           [ followed_by_line [ Re.str {|+++ b/|}; Re.group (Re.rep1 Re.notnl) ]
-           ; followed_by_line [ Re.str {|+++ |}; Re.group (Re.rep1 Re.notnl) ]
-           ]
+  let open Re in
+  let line xs = seq ((bol :: xs) @ [ eol ]) in
+  let filename = group @@ rep1 @@ compl [ space ] in
+  let timestamp = rep notnl in
```

Comment:
Can you convert this into matching a timestamp format or rename this into something that describes that this is just some leftovers data and we don't really care what it is?

In general it would be good to test this with paths that contain spaces. However that might be out of scope, given we don't support files with spaces anyway.

Reply:
> rename this into something that describes that this is just some leftovers data and we don't really care what it is?

Yeah that's a good point
> In general it would be good to test this with paths that contain spaces. However that might be out of scope, given we don't support files with spaces anyway.

Special characters in filenames are very finnicky and platform & tool dependent, I would much rather not chase a 100% full-compliance here.
I looked it up, and over at ocaml/opam-source-archives (the place they store the patches), there's over 300 patches, and not a single one has a file with a space in it.

Reply:
> rename this into something that describes that this is just some leftovers data and we don't really care what it is?

Done!